src/Files.Uwp/Views/MainPage.xaml

```diff
@@ -1,486 +1,450 @@
 ﻿<Page
-    x:Class="Files.Uwp.Views.MainPage"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:contract8Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,8)"
-    xmlns:controls="using:Files.Uwp.UserControls"
-    xmlns:converters="using:Files.Uwp.Converters"
-    xmlns:helpers="using:Files.Uwp.Helpers"
-    xmlns:converters1="using:Microsoft.Toolkit.Uwp.UI.Converters"
-    xmlns:custom="using:Microsoft.Toolkit.Uwp.UI.Controls"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:i="using:Microsoft.Xaml.Interactivity"
-    xmlns:icore="using:Microsoft.Xaml.Interactions.Core"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
-    xmlns:usercontrols="using:Files.Uwp.UserControls.MultitaskingControl"
-    xmlns:viewmodels="using:Files.Uwp.ViewModels"
-    muxc:BackdropMaterial.ApplyToRootOrPageBackground="False"
-    Background="Transparent"
-    KeyboardAcceleratorPlacementMode="Hidden"
-    Loaded="Page_Loaded"
-    SizeChanged="Page_SizeChanged"
-    NavigationCacheMode="Required"
-    mc:Ignorable="d">
-    <Page.DataContext>
-        <viewmodels:MainPageViewModel />
-    </Page.DataContext>
-    <Page.Resources>
-        <ResourceDictionary>
-            <ResourceDictionary.MergedDictionaries>
-                <ResourceDictionary Source="ms-appx:///ResourceDictionaries/DefaultGridSplitterStyle.xaml" />
-            </ResourceDictionary.MergedDictionaries>
+	x:Class="Files.Uwp.Views.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:contract8Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,8)"
+	xmlns:controls="using:Files.Uwp.UserControls"
```

Comment:
Spaces turned to tabs intentional?

Reply:
Yeah, btw, you can hide whitespaces in GitHub to make it easier to review.

Reply:
This `?w=1`? Yup been happier since I found out :)

Reply:
I think there is also a setting for it but that works